package.json

```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "@ant-design/icons": "^5.3.0",
     "@ant-design/pro-components": "^2.7.0",
+    "@nicholasadamou/react-iframe": "^1.0.4",
```

Comment:
this package is old, and doesn't have peer dependencies that specify the correct react version - and it doesn't seem to be used anywhere?

Reply:
removed